### PR TITLE
Player 3473

### PR DESCRIPTION
**Pull request by pilievOoyala**

### CODE REVIEW

File: src/main/js/main_html5.js [hunk 1]
```diff
@@ -936,13 +938,46 @@ require("../../../html5-common/js/utils/environment.js");
...
       raiseAudioChange(newTracks);
     };
 
+    /**
+     * Set the playback speed of the video element
+     * @public
+     * @method OoyalaVideoWrapper#setPlaybackSpeed
+     * @param  {number} speed The desired speed multiplier
+     */
+    this.setPlaybackSpeed = function(speed) {
```
Comment (aeng7) [inline src/main/js/main_html5.js]: Can you update the video plugin template with these new public functions? The template acts as part of our documentation for video plugins

Comment (pilievOoyala) [reply src/main/js/main_html5.js]: Will do.

Comment (pilievOoyala) [reply src/main/js/main_html5.js]: Added

File: src/main/js/main_html5.js [hunk 2]
```diff
@@ -936,13 +938,46 @@ require("../../../html5-common/js/utils/environment.js");
...
+      if (typeof speed !== 'number') {
```
Comment (jpfranco) [inline src/main/js/main_html5.js]: Same comment as in the other PR. We might need to check for `NaN`, etc.

File: tests/unit/main_html5/wrapper-api-tests.js [hunk 3]
```diff
@@ -1261,6 +1261,36 @@ describe('main_html5 wrapper tests', function () {
...
+
+    //speed should not be changed if you pass in bad values
+    wrapper.setPlaybackSpeed();
+    expect(wrapper.getPlaybackSpeed()).to.be(.5);
+    wrapper.setPlaybackSpeed(null);
+    expect(wrapper.getPlaybackSpeed()).to.be(.5);
+    wrapper.setPlaybackSpeed("1");
+    expect(wrapper.getPlaybackSpeed()).to.be(.5);
+    wrapper.setPlaybackSpeed({});
+    expect(wrapper.getPlaybackSpeed()).to.be(.5);
```
Comment (jpfranco) [inline tests/unit/main_html5/wrapper-api-tests.js]: Can we add a check for `NaN`?

**Comment by aeng7** [comment]
LGTM

**Pull request merged**